ieducar/intranet/empresas_cad.php

```diff
@@ -244,6 +254,16 @@ protected function validaCaracteresPermitidosComplemento()
 
     public function Editar()
     {
+        if (!empty($this->insc_est)) {
+            $inscricaoLimpa = trim(strtoupper($this->insc_est));
+            if ($inscricaoLimpa !== 'ISENTO' && !is_numeric(str_replace(['.', '-', ' '], '', $inscricaoLimpa))) {
```

Comment:
```suggestion
            if ($inscricaoLimpa !== 'ISENTO' && !ctype_digit($inscricaoLimpa)) {
```

Se puder avaliar, talvez dessa forma evitamo o replace para a validação.